codeReview/cr2/OOP_EX1/CalcAverage.cpp

```diff
@@ -0,0 +1,89 @@
+#include "string.h"
+#include "CalcAverage.h"
+#include <iostream>
+#include <cstring>
+#include <fstream>
+#include <string>
+#include <algorithm>
+#include <functional>
+using namespace std;
+int CalculateAverage::studentNum = 0;
+CalculateAverage::CalculateAverage(string filepath){
+	cout << "********----- Welcome to GPA calcualting program -----********" << '\n'<<endl;
+	ifstream myfile(filepath);
+	int id;
+	double phGrade, mathGrade, chGrade;
```

Comment:
```suggestion
	double phGrade, mathGrade, chGrade;
```
Не понятные переменные